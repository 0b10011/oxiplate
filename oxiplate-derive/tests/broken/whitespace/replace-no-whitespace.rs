use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline(r#"
{{ "replace" _}}{{_ "replace" }}
{{ "replace" _}}{{ "preserve" }}
{{ "preserve" }}{{_ "replace" }}
{# "replace" _#}{#_ "replace" #}
{# "replace" _#}{# "preserve" #}
{# "preserve" #}{#_ "replace" #}
"#)]
struct Data {}

fn main() {
    print!("{}", Data {});
}
