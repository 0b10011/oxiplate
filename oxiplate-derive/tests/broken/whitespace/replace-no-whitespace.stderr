error: Whitespace replace character `_` used before non-whitespace. Either add whitespace or change how whitespace is handled after this tag.
 --> tests/broken/whitespace/replace-no-whitespace.rs:5:14
  |
5 | {{ "replace" _}}{{_ "replace" }}
  |              ^^^

error: Whitespace replace character `_` used after non-whitespace. Either add whitespace or change how whitespace is handled before this tag.
 --> tests/broken/whitespace/replace-no-whitespace.rs:5:17
  |
5 | {{ "replace" _}}{{_ "replace" }}
  |                 ^^^

error: Whitespace replace character `_` used before non-whitespace. Either add whitespace or change how whitespace is handled after this tag.
 --> tests/broken/whitespace/replace-no-whitespace.rs:6:14
  |
6 | {{ "replace" _}}{{ "preserve" }}
  |              ^^^

error: Whitespace replace character `_` used after non-whitespace. Either add whitespace or change how whitespace is handled before this tag.
 --> tests/broken/whitespace/replace-no-whitespace.rs:7:17
  |
7 | {{ "preserve" }}{{_ "replace" }}
  |                 ^^^

error: Whitespace replace character `_` used before non-whitespace. Either add whitespace or change how whitespace is handled after this tag.
 --> tests/broken/whitespace/replace-no-whitespace.rs:8:14
  |
8 | {# "replace" _#}{#_ "replace" #}
  |              ^^^

error: Whitespace replace character `_` used after non-whitespace. Either add whitespace or change how whitespace is handled before this tag.
 --> tests/broken/whitespace/replace-no-whitespace.rs:8:17
  |
8 | {# "replace" _#}{#_ "replace" #}
  |                 ^^^

error: Whitespace replace character `_` used before non-whitespace. Either add whitespace or change how whitespace is handled after this tag.
 --> tests/broken/whitespace/replace-no-whitespace.rs:9:14
  |
9 | {# "replace" _#}{# "preserve" #}
  |              ^^^

error: Whitespace replace character `_` used after non-whitespace. Either add whitespace or change how whitespace is handled before this tag.
  --> tests/broken/whitespace/replace-no-whitespace.rs:10:17
   |
10 | {# "preserve" #}{#_ "replace" #}
   |                 ^^^
